x-pack/docs/en/rest-api/security/clear-roles-cache.asciidoc

```diff
@@ -27,8 +27,10 @@ For more information about the native realm, see
 [[security-api-clear-role-cache-path-params]]
 ==== {api-path-parms-title}
 
-`name`::
-  (string) The name of the role. 
+`<roles>`::
+(Required, string)
+Comma-separated list of roles to evict from the native role cache. To evict all
```

Comment:
Nit: I don't think we have the phrase "native role cache". There is only just one role cache. So I suggest
```suggestion
Comma-separated list of roles to evict from the role cache. To evict all
```